docs/guides/new_players/getting_started.mdx

```diff
@@ -39,7 +39,7 @@ Players can solve some of the NPC quests present after getting out of the vLAN,
 
 ### Obtaining T1 NPC locs
 
-Players can obtain NPC locs from solving the puzzles in public scripts which end in ".((%Lpublic%))", which are obtained from the aforementioned quests or from looking them up in FULLSEC sectors using [[((scripts.fullsec))]].
+Players can obtain NPC locs from solving the puzzles in public scripts which end in ".((%Lpublic%))", which are obtained from the aforementioned quests or from looking them up in FULLSEC sectors using (([[scripts.fullsec]])).
```

Comment:
```suggestion
Players can obtain NPC locs from solving the puzzles in public scripts which end in ".((%Lpublic%))", which are obtained from the aforementioned quests or from looking them up in FULLSEC sectors using [[scripts.fullsec:((scripts.fullsec))]].
```
Need an alias to accomplish what we want, could look into making this easier in the future since it seems like a common use case.